src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -156,6 +158,8 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-c-dropdown__toggle-text` | `<span>` | Defines the dropdown toggle text. **Required when text is present, adds truncation**. |
 | `.pf-c-dropdown__toggle-check` | `<label>` | Defines a checkbox in the toggle area of a split button dropdown. |
 | `.pf-c-dropdown__toggle-button` | `<button>` | Defines the toggle button for a split button dropdown. |
+| `.pf-c-dropdown__toggle-badge` | `<span>` | Defines the dropdown toggle badge. |
+| `.pf-c-dropdown__toggle-badge-text` | `<span>` | Defines the dropdown badge text. **Required**. |
```

Comment:
Do you still need these lines for `pf-c-dropdown__toggle-badge` and `pf-c-dropdown__toggle-badge-text`?